org.kde.okular.json

```diff
@@ -0,0 +1,235 @@
+{
+    "id": "org.kde.okular",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "okular",
+    "rename-icon": "okular",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "cleanup": [ 
+        "/include", 
+        "/lib/*.a", 
+        "/lib/*.la", 
+        "/lib/pkgconfig" 
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app --with-libraries=system",
+                "./b2 -j `nproc` install"
+            ],
+            "cleanup": [
+                "*.a"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/boost/files/boost/1.63.0/boost_1_63_0.tar.bz2",
+                    "sha256": "beae2529f759f6b3bf3f4969a19c2e9d6f0c503edcb2de4a61d1428519fcb3b0"
+                }
+            ]
+        },
+        {
+            "name": "kjs",
+            "config-opts": ["-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_INSTALL_LIBDIR=lib"],
```

Comment:
`-DCMAKE_INSTALL_LIBDIR=lib` isn't needed any more?

Reply:
You're right, I should do a pr to the flatpak-kde-applications repo as well.

Reply:
For projects that use https://github.com/KDE/extra-cmake-modules` (like all of KDE) `-DCMAKE_INSTALL_LIBDIR=lib` is still needed.

Reply:
@eszlari do you know what's the issue? I could look into it if you can tell me what's going wrong.

Reply:
I don't know what the cause is, but without it libs get installed under `lib64`. See my last comment under https://github.com/flatpak/freedesktop-sdk-base/issues/15